### ISSUE TITLE
Firestore integration via cocoapods is broken

### ISSUE THREAD

**Issue by aplusm** [firebase/firebase-ios-sdk#736]
**Describe your environment**

  * Xcode version: 9.2
  * Firebase SDK version: 4.8.2
  * Firebase Component: Firestore
  * Component version: 0.10.0

**Describe the problem**

Simple project with a basic Podfile:

```
platform :ios, '9.0'
use_frameworks!

target 'test' do
  pod 'Firebase/Core'
  pod 'Firebase/Firestore'
end
```

When building, the following error is detected:

```
**Lexical or Preprocessor Issue**

[ProjectDir]/Pods/gRPC-Core/src/core/ext/filters/client_channel/lb_policy/grpclb/proto/grpc/lb/v1/load_balancer.pb.h:6:10: 'third_party/nanopb/pb.h' file not found

 In file included from [ProjectDir]/Pods/gRPC-Core/src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc:100

In file included from [ProjectDir]/Pods/gRPC-Core/src/core/ext/filters/client_channel/lb_policy/grpclb/load_balancer_api.h:25

```

**Comment by aplusm** [comment]
Content of Podfile.lock : 

PODS:
  - BoringSSL (9.2):
    - BoringSSL/Implementation (= 9.2)
    - BoringSSL/Interface (= 9.2)
  - BoringSSL/Implementation (9.2):
    - BoringSSL/Interface (= 9.2)
  - BoringSSL/Interface (9.2)
  - Firebase/Core (4.8.2):
    - FirebaseAnalytics (= 4.0.9)
    - FirebaseCore (= 4.0.14)
  - Firebase/Firestore (4.8.2):
    - Firebase/Core
    - FirebaseFirestore (= 0.10.0)
  - FirebaseAnalytics (4.0.9):
    - FirebaseCore (~> 4.0)
    - FirebaseInstanceID (~> 2.0)
    - GoogleToolboxForMac/NSData+zlib (~> 2.1)
    - nanopb (~> 0.3)
  - FirebaseCore (4.0.14):
    - GoogleToolboxForMac/NSData+zlib (~> 2.1)
  - FirebaseFirestore (0.10.0):
    - FirebaseAnalytics (~> 4.0)
    - FirebaseCore (~> 4.0)
    - gRPC-ProtoRPC (~> 1.0)
    - leveldb-library (~> 1.18)
    - Protobuf (~> 3.1)
  - FirebaseInstanceID (2.0.8):
    - FirebaseCore (~> 4.0)
  - GoogleToolboxForMac/Defines (2.1.3)
  - GoogleToolboxForMac/NSData+zlib (2.1.3):
    - GoogleToolboxForMac/Defines (= 2.1.3)
  - gRPC (1.8.6):
    - gRPC-RxLibrary (= 1.8.6)
    - gRPC/Main (= 1.8.6)
  - gRPC-Core (1.8.6):
    - gRPC-Core/Implementation (= 1.8.6)
    - gRPC-Core/Interface (= 1.8.6)
  - gRPC-Core/Implementation (1.8.6):
    - BoringSSL (~> 9.0)
    - gRPC-Core/Interface (= 1.8.6)
    - nanopb (~> 0.3)
  - gRPC-Core/Interface (1.8.6)
  - gRPC-ProtoRPC (1.8.6):
    - gRPC (= 1.8.6)
    - gRPC-RxLibrary (= 1.8.6)
    - Protobuf (~> 3.0)
  - gRPC-RxLibrary (1.8.6)
  - gRPC/Main (1.8.6):
    - gRPC-Core (= 1.8.6)
    - gRPC-RxLibrary (= 1.8.6)
  - leveldb-library (1.20)
  - nanopb (0.3.8):
    - nanopb/decode (= 0.3.8)
    - nanopb/encode (= 0.3.8)
  - nanopb/decode (0.3.8)
  - nanopb/encode (0.3.8)
  - Protobuf (3.5.0)

DEPENDENCIES:
  - Firebase/Core
  - Firebase/Firestore

SPEC CHECKSUMS:
  BoringSSL: f3d6b8ce199b9c450a8cfc14895d07a2627fc232
  Firebase: 7d3b8cd837ad9fcd391657734c0d56dab8e9a5a3
  FirebaseAnalytics: 388b630c15713f5dbf364071f5f3d6077fb52f4e
  FirebaseCore: 2e0b98fb2d64ca8140136beff15772bdd14d2dd7
  FirebaseFirestore: 713f0c555e7af5ac03d0fec0e2477c48857f4977
  FirebaseInstanceID: 81df5805a08001e69138664bdd02c6719a9ac80f
  GoogleToolboxForMac: 2501e2ad72a52eb3dfe7bd9aee7dad11b858bd20
  gRPC: 40acd783b734b4d5fc012713090fc92166471fb8
  gRPC-Core: d58a324ac8889bcc837dbfed4224ef718036926e
  gRPC-ProtoRPC: 7f3da3b3a90ae851f774c4eec138a9428d54791b
  gRPC-RxLibrary: c1a5f74a2a54022fc386e09fcdd76fc8febd73aa
  leveldb-library: 08cba283675b7ed2d99629a4bc5fd052cd2bb6a5
  nanopb: 5601e6bca2dbf1ed831b519092ec110f66982ca3
  Protobuf: 8a9838fba8dae3389230e1b7f8c104aa32389c03

PODFILE CHECKSUM: 466f297110c647c57ef654ba2f83305931e09eca

COCOAPODS: 1.4.0

**Comment by wilhuff** [comment]
I can't reproduce this with
```
platform :ios, '9.0'
use_frameworks!

target 'test2' do
  pod 'Firebase/Core'
  pod 'Firebase/Firestore'
end
```
Also Xcode 9.2, Cocoapods 1.4.0. The resulting Podfile.locks are essentially the same.

Looking at load_balancer.pb.h:6:10 I see this line:
```c
#include <nanopb/pb.h>
```
Which doesn't mention `third_party/nanopb/pb.h`. What does yours look like?

`pb.h` should be coming from `Pods/nanopb/pb.h`. Can you confirm this file exists?

**Comment by aplusm** [comment]
Thank you @wilhuff for your reply.

I just solved the issue by myself. 

Some Firestore dependencies pods (gRPC*, BoringSSL) were not installed properly by cocoapods due to a **sed error** .
I'm using GNU sed on my machine (installed through homebrew) and this breaks the install of those pods.

I found this using cocoapods in verbose mode, there were no feedback for this error when running in regular mode.
